package.json

```diff
@@ -84,7 +84,7 @@
     "sanitize-html": "1.19.1",
     "showdown": "1.9.0",
     "slugify": "1.3.3",
-    "styled-components": "3.4.10",
+    "styled-components": "4.1.1",
```

Comment:
4.1.2 is the latest now

Reply:
The world moves fast. I'll update it to the latest if you think we should.